operations/flags.go

```diff
@@ -100,7 +100,6 @@ func restServiceFlags(flags ...cli.Flag) []cli.Flag {
 		cli.IntFlag{
 			Name:  clientPortFlag,
 			Usage: "port for the remote cedar service.",
-			Value: 3000,
```

Comment:
why remove this?

Reply:
This is mostly a problem of what a sane default looks like. For a localhost service, it's fine, but if you want to use it as a client to production cedar, you have to specify `--host https://cedar.mongodb.com --port 0`, which is kind of weird.

Reply:
Oh I thought remote used port 3000 for some reason...

Reply:
Does the value not have to be set to 0 then?

Reply:
I changed the `BeforeFunc` a bit to handle default vs non-default behavior. It sets the port to 3000 if you don't specify the host flag (so using the default values will still connect to the default local cedar). However, if you set host (e.g. `--host=https://cedar.mongodb.com`), the port will not be set to any default (so you don't need to specify `--port=0`).